src/lib/components/Creatibutors/CreatibutorsModal.js

```diff
@@ -229,6 +230,81 @@ export class CreatibutorsModal extends Component {
       };
     });
 
+    results.push({
+      text: 'Manual entry',
+      value: 'Manual entry',
+      extra: 'Manual entry',
+      key: 'manual-entry',
+      content: (
+          <Header>
+            <Header.Content>
+              <p>Couldn't find your person? You can <a>create a new entry</a></p>
+            </Header.Content>
+          </Header>
+      ),
+    });
+    return results;
+  }
+
+  onValueChange = ({ event, data, formikProps }, selectedSuggestions) => {
```

Comment:
```suggestion
  onPersonSearchChange = ({ event, data, formikProps }, selectedSuggestions) => {
```

To avoid possible name clashes.